sdk/streamlib-python-wheel/tests/test_compute_kernel.py

```
@@ -139,23 +143,69 @@ def test_a_binding_naming_an_unknown_surface_is_refused(start_app_under_test):
-        "a device texture has no mapping in this process, so reading its "
-        "pixels here must refuse rather than answer"
+    assert observed["published_pixel"] == list(FILLED_SOURCE_RGBA), (
+        "the staged edit must publish into the surface's own allocation at the "
+        f"block edge: {observed['published_pixel']!r}"
+    )
+
+    # The shader writes `1.0 - source.rgb` with a zero bias and passes alpha
+    # through; unorm round-trip is worth one code point of slack, no more.
+    inverted = [255 - channel for channel in FILLED_SOURCE_RGBA[:3]]
+    read_back = observed["kernel_output_pixel"]
+    assert all(abs(read - want) <= 1 for read, want in zip(read_back[:3], inverted)), (
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)